configs/sysbench_memory_example.yaml

```diff
@@ -1,7 +1,8 @@
 threads: 5
 events: 0
 time: 15
-memory-block-size: '2KiB'
-memory-total-size: '100GiB'
-memory-scope: 'local'
-memory-oper: 'write'
\ No newline at end of file
+memoryblocksize: '2KiB'
+memorytotalsize: '100GiB'
+memoryscope: 'local'
+memoryoperation: 'write'
+operation: 'memory'
```

Comment:
As noted above, this key is no longer used by the schema.

Reply:
> Thanks for catching the inaccuracy in the README file!
> 
> Unfortunately your changes to the example input files are outdated. Are you perhaps working with an older version of the plugin? You can always get our latest main and release builds of the plugin containers at https://quay.io/repository/arcalot/arcaflow-plugin-sysbench?tab=tags
> 
> If you are building from source, make sure you are on the main branch or a release tag.

Yup, my bad, mistyped the name when building and used and older release that was from my previous testing. Removed the config file changes and only have the doc updates.